io.github.ec_.Quake3e.OpenArena.yaml

```diff
@@ -0,0 +1,73 @@
+modules:
+  # CHANGED: Module name isn't displayed to users and convention says lowercase
+  - name: openarena
+    buildsystem: simple
+    sources:
+      # CHANGED: Pull direct from the upstream source
+      - type: git
+        url: "https://github.com/ec-/Quake3e"
+        commit: "f69865508ef4f3d63bcc82c8505d661a1c290f92"
+        tag: "2022-10-14"
+        x-checker-data: # Allow bot to automatically make PRs for new releases
+          type: git
+          tag-pattern: "^(\\d{4}-\\d{2}-\\d{2})$"
+    # CHANGED: Removed config-opts because it's for ./configure, not bare make
+    build-commands:
+      - make USE_CODEC_VORBIS=1 USE_INTERNAL_OGG=1 USE_VOIP=1 USE_CODEC_OPUS=1 USE_OPENAL=1 USE_MUMBLE=1 USE_FREETYPE=1 USE_INTERNAL_LIBS=1 BUILD_SERVER=0COPYDIR=/app/share/openarena COPYBINDIR=/app/bin DEFAULT_BASEDIR=/app/share/openarena USE_CURL=1 USE_CURL_DLOPEN=0
+      - make install "DESTDIR=${FLATPAK_DEST}/bin"
+      # CHANGED: Use ${FLATPAK_DEST}/bin because Quake3e is one of those weird
+      #          exceptions that doesn't follow platform conventions.
+  - name: openarena-data
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: "https://downloads.sourceforge.net/project/oarena/openarena-0.8.8.zip"
+        sha256: "5a8faf7f5b51f351b0a1618c06b6b98a5f1a6758f1d39818de2c87df2a0bac4a"
+    build-commands:
+      - mv baseoa missionpack /app/bin/
+  - name: launch-script
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: "https://github.com/xplshn/openarena/blob/FlatHub-complaint/launch.zip?raw=true"
+        sha256: "898cd1a4bd453609007ef1aef5ef3b03f9b44cadb9048d33c9fd5f418ad0d75a"
+    build-commands:
+      - mv launch /app/bin/
+      - chmod u+x /app/bin/launch
+      - install -Dm644 io.github.ec_.Quake3e.OpenArena.png -t /app/share/icons/hicolor/128x128/apps
+      - install -Dm644 io.github.ec_.Quake3e.OpenArena.desktop -t /app/share/applications/
+      - install -Dm644 io.github.ec_.Quake3e.OpenArena.appdata.xml -t /app/share/appdata/
+cleanup:
+  - "/include"
+  - "/lib/pkgconfig"
+  - "/share/aclocal"
+  - "/share/man"
+  - "*.la"
+  - "*.a"
+finish-args:
+  # X11 + XShm access + Wayland
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=all
+  - --socket=pulseaudio # CHANGED: Added for sound support
+  # Needs to talk to the network:
+  - --share=network
+  # CHANGED: Needs a persistent location for Quake 3 data files the user
+  # has permission to add things to and this is the only one in Quake3e's
+  # search path.
+  - --persist=.q3a
+  # CHANGED: Removed --filesystem=xdg-data because the Flathub submission
+  #          guides explicitly tell you NOT to try to preserve access to the
+  #          un-sandboxed versions of paths like ~/.local/share. Users can
+  #          copy/move their stuff to ~/.var/app/org.fxt.openarena/data
+  #          manually if they want to.
+post-install:
+  - mv /app/bin/openarena.* /app/bin/openarena
```

Comment:
What's the point of this line? There's no file matching `openarena.*` in /app/bin.

Reply:
flatpak-builder says:

```
(flatpak-builder:2): flatpak-builder-WARNING **: 09:28:00.810: Unknown property post-install for type BuilderManifest
```

Reply:
> What's the point of this line? There's no file matching `openarena.*` in /app/bin.

That's what the OpenARENA on flathub has in its manifiest, I also didn't know why but I wanted to make it a drop-in replacement

Reply:
that doesn't make it right. It's invalid here. flatpak-builder tells you.

Reply:
Updated: this is not even what openarena do.

Reply:
Well, its manifiest has that line, I copied the post-install args

Reply:
post-install in that context is not valid. flatpak-builder complains about it, I just which it was an error.

This is not what openarena does.

Reply:
https://github.com/flathub/ws.openarena.OpenArena/blob/master/ws.openarena.OpenArena.json
It is, look  "post-install" section brackets

Reply:
> post-install in that context is not valid. flatpak-builder complains about it, I just which it was an error.
> 
> This is not what openarena does.

Anyway, since its a bit cryptic and weird, I think I'll remove it because it is not trustworthy to have that in your recipe